cmd/thanos/sidecar.go

```diff
@@ -111,6 +113,17 @@ func runSidecar(
 		maxt: math.MaxInt64,
 	}
 
+	confContentYaml, err := objStoreConfig.Content()
+	if err != nil {
+		return errors.Wrap(err, "")
```

Comment:
This is really wrong, what's the of empty wrap ?

Reply:
My fault. Miss is, IMO this wrap is not need here.

Reply:
Could we wrap it adding more context? some thinkg `errors.wrap(err, "error getting object store config")`

Reply:
Sure. But all other  code of objStoreConfig.Content  just return err directly. please consider add this wrap later.